toolkit/scripts/pkggen.mk

```diff
@@ -177,10 +177,6 @@ clean-compress-srpms:
 	rm -rf $(srpms_archive)
 
 ifeq ($(REBUILD_PACKAGES),y)
-# If we are responsible for building a toolchain, make sure those RPMs are also present in the output directory
-ifeq ($(REBUILD_TOOLCHAIN),y)
-$(RPMS_DIR): $(toolchain_rpms)
-endif
```

Comment:
Why we removed this dependency? Don't we want to have the toolchain in the `out/RPMS` directory if we're asking to rebuild it? Or is this already happening through other dependencies?

Reply:
Moved to https://github.com/microsoft/CBL-Mariner/pull/4210/files#diff-eef96fc8d3aac83583f5507a002236b9b08b6ce1d2fd7d9f0f04b06bd84d6a7eR57